From 24c29d4c8c4854c42829abb21df755ed2b5a3729 Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@cloudera.com>
Date: Tue, 6 Oct 2015 17:26:03 -0700
Subject: [PATCH 0916/2848] CLOUDERA-BUILD. Revert "HADOOP-12451. Setting
 HADOOP_HOME explicitly should be allowed.
 (kasha)"

This reverts commit 3023ed83fc7c26fdef77a3a9bc46b714fe5a8de5.

Change-Id: I6457155b479905a56a6816de7c7bad576aba004e
---
 .../hadoop-common/src/main/bin/hadoop-config.sh    |    7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/bin/hadoop-config.sh b/hadoop-common-project/hadoop-common/src/main/bin/hadoop-config.sh
index c813e7f..15e7afa 100644
--- a/hadoop-common-project/hadoop-common/src/main/bin/hadoop-config.sh
+++ b/hadoop-common-project/hadoop-common/src/main/bin/hadoop-config.sh
@@ -233,11 +233,8 @@ TOOL_PATH="${TOOL_PATH:-$HADOOP_PREFIX/share/hadoop/tools/lib/*}"
 
 HADOOP_OPTS="$HADOOP_OPTS -Dhadoop.log.dir=$HADOOP_LOG_DIR"
 HADOOP_OPTS="$HADOOP_OPTS -Dhadoop.log.file=$HADOOP_LOGFILE"
-
-if [ "$HADOOP_HOME" = "" ]; then
-  HADOOP_HOME=$HADOOP_PREFIX
-fi
-if [ "$cygwin" = true ]; then
+HADOOP_HOME=$HADOOP_PREFIX
+if $cygwin; then
   HADOOP_HOME=$(cygpath -w "$HADOOP_HOME" 2>/dev/null)
 fi
 export HADOOP_HOME
-- 
1.7.9.5

